playbooks/roles/ses-rook/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+ses_rook_repos:
```

Comment:
Could you document the different variables so it's more clear what these variables are good for?

Reply:
really? I think its kind of clear with their names. Which one is not clear enough to what it refers?

Reply:
- `ses_rook_path_in_workers` ? Which workers? CaaSP workers? SES OSD`s ?
- `ses_rook_manifests_dir` ? What manifests? For helm maybe?
- `ses_rook_package` ? I guess the RPM package name that needs to be installed on the deployer to have the manifests available?

Reply:
I....I do not agree here but will try to make them clearer. Dont blame me if we end up with a 80 chars var thought, Im terrible at naming stuff!

Reply:
Hope they are clearer now?

Reply:
I was more thinking about commenting the variable instead of changing the name - anyway - looks better imo :)